source/schema.txt

```diff
@@ -109,8 +109,8 @@ For fields that contain multiple data types,
 |compass-short| displays a percentage breakdown of
 the various data types across documents. In the example below,
 the chart shows the contents of a field called ``phone_no`` in which
-81% of documents are of type ``string``, and the remaining 19% are of
-type ``number``.
+20% of documents are of type ``int32``, and the remaining 80% are of
```

Comment:
Changed this since the updated UI makes the percentage cover the collection name. Also, the ``number`` type doesn't exist anymore — it's now ``int32`` (or ``int64``)